sag_et.par

```diff
@@ -4,195 +4,253 @@
+/|  |)  /|)  /|\  (|\  ||)  ||\  /||\
 d 94 55
 s 47
-~|(  /|  (|(  /|\  (|)  ~N(  N\  (N(  /N\
+~|(  /|  (|(  /|\  (|)  ~||(  ||\  (||(  /||\
 d 96 56
 s 32
-/|  |~  |\  /|\  /N  /N~  N\  /N\
+/|  |~  |\  /|\  /||  /||~  ||\  /||\
 d 99 58
 s 33
-~|  /|  ~|)  //|  )/|\  )N(  ~~N  N\  )//N  /N\
+~|  /|  ~|)  //|  )/|\  )||(  ~~||  ||\  )//||  /||\
 d 106 62
-)|(  /|  (|  //|  /|\  )N(  )N~  N\  //N  /N\
+)|(  /|  (|  //|  /|\  )||(  )||~  ||\  //||  /||\
 d 108 63
 s 54
-)/|  (|  |)  /|)  (|\  N)  )N~  )N)  /N\
+)/|  (|  |)  /|)  (|\  ||)  )||~  )||)  /||\
 d 111 65
 s 37
-)|(  /|  |\  //|  /|\  (|)  )N(  /N  N\  //N  /N\
+)|(  /|  |\  //|  /|\  (|)  )||(  /||  ||\  //||  /||\
 d 118 69
 s 59
-)|(  /|  |)  //|  /|\  (|)  )N(  N)  N\  //N  /N\
+)|(  /|  |)  //|  /|\  (|)  )||(  ||)  ||\  //||  /||\
 d 120 70
-)/|  (|  |)  /|~  /|\  N~  N)  )N~  )N)  /N\
+)/|  (|  |)  /|~  /|\  ||~  ||)  )||~  )||)  /||\
 d 125 73
-|(  /|  |)  //|  /|)  (|\  )N(  N)  N\  /N)  /N\
+|(  /|  |)  //|  /|)  (|\  )||(  ||)  ||\  /||)  /||\
 d 130 76
-|(  /|  |)  |\  /|)  /|\  (|\  /N  N)  N\  /N)  /N\
+|(  /|  |)  |\  /|)  /|\  (|\  /||  ||)  ||\  /||)  /||\
 d 132 77
 s 66
-~|(  (|  |~  |\  /|)  (|\  /N  /N~  )N~  (N(  /N\
+~|(  (|  |~  |\  /|)  (|\  /||  /||~  )||~  (||(  /||\
 d 142 83
 s 71
-|(  /|  |)  |\  /|)  /|\  (|)  (|\  /N  N)  N\  /N)  /N\
+|(  /|  |)  |\  /|)  /|\  (|)  (|\  /||  ||)  ||\  /||)  /||\
 d 144 84
-~|(  /|  |~  |)  /|~  /|\  N~  N)  /N~  N\  (N(  /N\
+~|(  /|  |~  |)  /|~  /|\  ||~  ||)  /||~  ||\  (||(  /||\
 d 152 89
 s 76
 )|(  ~|(  /|  |\  (|(  //|  /|\  (|)
-  )N(  ~N(  /N  N\  (N(  //N  /N\
+  )||(  ~||(  /||  ||\  (||(  //||  /||\
 d 156 91
 s 52
 ~|(  /|  |~  |)  |\  /|)  (|\
-  /N  N)  /N~  N\  (N(  /N\
+  /||  ||)  /||~  ||\  (||(  /||\
 d 159 93
 |(  ~|(  /|  |)  (|(  //|  /|\  (|)
-  )N(  ~N(  N)  N\  (N(  /N)  /N\
+  )||(  ~||(  ||)  ||\  (||(  /||)  /||\
 d 166 97
 s 83
 |(  )|(  /|  |)  (|(  //|  /|\  (|)
-  )N(  ~N(  N)  N\  //N  /N)  /N\
+  )||(  ~||(  ||)  ||\  //||  /||)  /||\
 d 168 98
 ~|(  /|  (|  |)  |\  /|)  /|\  (|\
-  /N  N)  )N~  N\  (N(  /N\
+  /||  ||)  )||~  ||\  (||(  /||\
 d 171 100
 s 57
 |(  ~|(  /|  |)  |\  //|  /|)  /|\  (|\
-  )N(  /N  N)  N\  (N(  /N)  /N\
+  )||(  /||  ||)  ||\  (||(  /||)  /||\
 d 176 103
 s 44 88
 |(  ~|(  /|  |)  |\  (|(  /|)  /|\  (|)  (|\
-  ~N(  /N  N)  N\  (N(  /N)  /N\
+  ~||(  /||  ||)  ||\  (||(  /||)  /||\
 d 180 105
 s 45 90
 ~|(  /|  (|  |~  |)  /|~  /|)  (|\
-  N~  N)  /N~  )N~  N\  (N(  /N\
+  ||~  ||)  /||~  )||~  ||\  (||(  /||\
 d 183 107
 s 61
 |(  ~|(  /|  |)  |\  (|(  /|)  /|\  (|)  (|\
-  ~N(  /N  N)  N\  (N(  /N)  /N\
+  ~||(  /||  ||)  ||\  (||(  /||)  /||\
 d 192 112
 ~|(  /|  (|  |~  |)  |\  /|)  /|\  (|\
-  /N  N)  /N~  )N~  N\  (N(  /N\
+  /||  ||)  /||~  )||~  ||\  (||(  /||\
 d 198 116
 )|(  |~  ~~|  /|  |\  ~|)  (|(  //|  /|\  )/|\  (|)
-  )N(  ~N(  ~~N  /N  N\  ~N)  /N~  //N  /N\
+  )||(  ~||(  ~~||  /||  ||\  ~||)  /||~  //||  /||\
+d 204 119

```

Comment:
I wasn't sure why the remaining Trojan notations above 192 that were not visualized in the Xenharmonikon article but which are able to be induced from the material there (204, 216, 228, 240) were not included here, so I added them.

Reply:
That's probably OK. Although there were some issues with them that I should check. See the spreadsheet with 240 in its name here: http://forum.sagittal.org/viewtopic.php?p=1726#p1726

Reply:
I see. Yes, you indicate that the 228-EDO notation has a flag arithmetic problem with (|\ not equaling (| + |\. Indeed 12 ≠ 4 + 7. That's the only obvious issue I see. Is this something you want to try to address now? Let me know if you'd rather keep 228 out of sag_et.par for now.

Reply:
Actually, I sent you to the wrong spreadsheet. That one was an experiment to see what would happen if the boundaries were evenly-spaced, on odd multiples of 2.5 cents. It doesn't work. It breaks the important 72edo notation.

The other spreadsheet has the correct boundaries and shows bad flag arithmetic for both 216 and 228. But they can't be fixed without breaking other things. The boundaries are already the best they can be. So please keep the notations you have for 204, 216, 228 and 240.

Reply:
Right-o.